ushahidi/platform-client-mzima
# [Outreachy Task Submission] Improve text description for number of posts in collection

### Description
This pull request addresses issue ushahidi/platform#4727 for improving the text description for the number of posts displayed in each collection. Previously, only the number of posts was shown, which could be unclear to users.

### Changes Made:
- Added descriptive text to indicate the number of posts available in each collection.
- Modified styles to ensure proper alignment and visual presentation of the text description.

### Testing Checklist:
1. Start the mobile app with `npm run mobile:serve`.
2. Select the deployment (e.g., accessibility_ushahidi).
3. Click on the profile icon and either log in or sign up.
4. Click on the profile icon again and select "Collection".
5. Verify that you see a list of collections with the associated number of posts in each collection.
6. Confirm that the added descriptive text for the number of posts is displayed correctly.
7. Ensure proper alignment and visual presentation of the text description.

https://github.com/ushahidi/platform-client-mzima/assets/68381641/20856167-59b5-420b-8568-2c8290fba345
